### ISSUE TITLE
Simplify quickClean message

### ISSUE THREAD

**Issue by pStyl3** [loot/skyrimse#1280]
**&quickClean**
> Due to Improvements made to the cleaning process with the official release of xEdit 4.0.x+, any mods which were verified as clean with older versions of xEdit may now require additional cleaning. A guide for running xEdit 4.0.x+ in **Quick Auto Clean** mode can be found [here](https://tes5edit.github.io/docs/6-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).

A lot of time has passed since @ElminsterAU introduced the new QAC process to clean plugins. I think that the message could be changed to something more generic and that it may no longer be important to indicate that QAC may find more than the old process of cleaning plugins. QAC is the de facto standard now. So the question is, what should we change it to?

For reference, the old **&dirtyPlugin** message, which was used before QAC was introduced (and still is used in the Skyrim masterlist):
> An explanation of Dirty Edits is available [here](https://tes5edit.github.io/docs/6-mod-cleaning-and-error-checking.html#AppendixA).

First of all, the url in quickClean should be changed to this one (notice that the number 6 changed to 7):
`https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods`

Here is a possible new message:
> The cleaning process Quick Auto Clean of xEdit has found, that this plugin may require cleaning. A guide for running xEdit 4.0.x+ in **Quick Auto Clean** mode can be found [here](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).

Or a more direct variant:
> The cleaning process Quick Auto Clean of xEdit has found, that this plugin requires cleaning. A guide for running xEdit 4.0.x+ in **Quick Auto Clean** mode can be found [here](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).

I'd like to hear other opinions on this, before deciding the final version of this message. Maybe something entirely different than the two examples above is suited better?

As always, we need as many translations for this message as possible, especially in this important case:
- [ ] English (en)
- [ ] Czech (cs)
- [ ] Danish (da)
- [ ] German (de)
- [ ] Spanish (es)
- [ ] Finnish (fi)
- [ ] French (fr)
- [ ] Japanese (ja)
- [ ] Korean (ko)
- [ ] Polish (pl)
- [ ] Portuguese (pt)
- [ ] Portuguese Brazilian (pt_BR)
- [ ] Russian (ru)
- [ ] Swedish (sv)
- [ ] Chinese (zh_CN)

**Comment by Ortham** [comment]
The `&quickClean` message is appended to the generated message ("X found Y", where Y can be "dirty edits" or a breakdown of the different counts of dirty edits found), so I don't think it's beneficial to repeat that dirty edits have been found. I suggest

> A guide to cleaning plugins using xEdit can be found [here](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).

I'm not sure if it's more or less useful to mention Quick Auto Clean in the message, or leave that to the xEdit guide.

**Comment by pStyl3** [comment]
> The &quickClean message is appended to the generated message ("X found Y", where Y can be "dirty edits" or a breakdown of the different counts of dirty edits found), so I don't think it's beneficial to repeat that dirty edits have been found. I suggest

I agree, it would be redundant. Your suggestion is fine for me, it's even more compact and does what it needs to do, it links the user to a cleaning guide.

> I'm not sure if it's more or less useful to mention Quick Auto Clean in the message, or leave that to the xEdit guide.

After thinking about it once more, I would also say, that we don't necessarily need to mention QAC in that message. After all, the process of using QAC is pretty straight forward and it is clearly described in the linked guide. Even more so, if we think of QAC as the new standard to clean plugins, it no longer needs to be mentioned seperately or to be explained in a special way. It is, what it is, _the_ way to clean plugins.

**Comment by pStyl3** [comment]
Okay, let's settle on @Ortham's suggestion:
`A guide to cleaning plugins using xEdit can be found [here](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).`

**German:**
`Eine Anleitung zum Säubern von Plugins mit xEdit finden Sie [hier](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).`

Would the following people be able to help with translating that sentence? Posting the translated sentence in this issue would be sufficient.

@tktk11 Japanese (ja)
@sean-kang Korean (ko)
@Yoosk Polish (pl)
@ironmagician Portuguese (pt)
@TokcDK Russian (ru)
@simoneddeland Swedish (sv)

Thanks a lot to everyone who finds the time to help!

**Comment by ironmagician** [comment]
Hello, here's the sentence translated:

**Portuguese (pt)**:
`Um guia para limpar plugins utilizando xEdit pode ser encontrado [aqui](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods)`

Feel free to use it as 'pt-br' translation also, if no 'pt-br' appears.

**Comment by gnunesm** [comment]
I was going to contribute a 'pt-br' version, but in this case it really is the same as @ironmagician 's translation.

**Comment by pStyl3** [comment]
Alright, thank you both @ironmagician and @gnunesm.

**Comment by tktk11** [comment]
Hi, I translate it.
**Japanese (ja):**
`xEditを使ったプラグインのクリーニングのガイドは[こちら](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods)にあります。`

**Comment by TokcDK** [comment]
Here is Russian (ru):
`Гид по очистке с помощью xEdit можно найти [здесь](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).`

**Comment by pStyl3** [comment]
Thanks a lot @tktk11 and @TokcDK.

**Comment by simoneddeland** [comment]
Swedish (sv):
`En guide till att rensa plugins med xEdit finns [här](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).`

**Comment by pStyl3** [comment]
@3ventic Would you mind translating this one sentence as well? So:
`A guide to cleaning plugins using xEdit can be found [here](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).`

**Comment by pStyl3** [comment]
@RacerBG Since you added the Bulgarian translation for the LOOT application itself (will be added with the next official release), would you mind also translating some general messages for our masterlists? That way, once the next version will be released, those messages will be displayed in Bulgarian, too.

For one, there is this one sentence that you can see here in the post above or in the original post at the start of this topic.
On the other hand there is issue number [646](https://github.com/loot/skyrimse/issues/646) which holds additional messages.
PRs are welcome, but not strictly necessary, you could just post the translations in that issue, if you want.

**Comment by 3ventic** [comment]
> @3ventic Would you mind translating this one sentence into Finnish as well? So:
> `A guide to cleaning plugins using xEdit can be found [here](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).`

`Ohjeet liitännäisten puhdistamiseen xEditillä löytyvät [tästä linkistä](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).

LMK if you prefer a PR.

**Comment by pStyl3** [comment]
@3ventic Thanks for that. No need for a PR really, I will handle the rest!

**Comment by sibir-ine** [comment]
@pStyl3 A friend who'd like to remain anonymous has provided a Simplified Chinese translation. I'll add it to the masterlists shortly, but how should we credit him? Something like "anonymous translator"? He & possibly a few others will provide more translations after we refactor the messages a bit more.

**Comment by pStyl3** [comment]
@sibir-ine If he or she prefers to stay anonymous, we should handle it just like that and not add a mentioning of any sort. Be sure to let him/her know our gratitude though.

**Comment by RacerBG** [comment]
> @RacerBG Since you added the Bulgarian translation for the LOOT application itself (will be added with the next official release), would you mind also translating some general messages for our masterlists? That way, once the next version will be released, those messages will be displayed in Bulgarian, too.
> 
> For one, there is this one sentence that you can see here in the post above or in the original post at the start of this topic.
> On the other hand there is issue number [646](https://github.com/loot/skyrimse/issues/646) which holds additional messages.
> PRs are welcome, but not strictly necessary, you could just post the translations in that issue, if you want.

@pStyl3 Here you go (Bulgarian translation):

`Информация за почистване на приставките в xEdit може да намерите [тук](https://tes5edit.github.io/docs/7-mod-cleaning-and-error-checking.html#ThreeEasyStepstocleanMods).`
